app/src/api/learningJourneyCourse.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import { LEARNING_JOURNEY_COURSE_ENDPOINT } from "./config";
+
+const axiosLJCourseInstance = axios.create({
+  baseURL: LEARNING_JOURNEY_COURSE_ENDPOINT,
+  timeout: 5000,
+  headers: { "X-Custom-Header": "foobar" },
+});
+
+
+export const deleteLearningJourneyCourseWithLJId = async (ljId) => {
+  try {
+    const res = await axiosLJCourseInstance.delete(`all/lj_id=${ljId}`);
```

Comment:
Is there an endpoint that takes in the path `all/lj_ud=${ljId}` here? I don't remember the fastapi endpoints to be formatted with a query param looking path variable but without the `?` notation